src/Main.java

```diff
@@ -1,5 +1,82 @@
+import java.util.Random;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        task1();
+
     }
+    // Объявить метод «сгенерироватьМассив»
+    public static int[] generateRandomArray() {
+        Random random = new Random();
+        int[] arr = new int[30];
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+        return arr;
+    }
+    public static void task1() {
```

Comment:
```suggestion
    public static void tasks() {
```
почему task1, если там внутри все 4 задачи?